deploy/kubeturbo-for-openshift/README.md

```diff
@@ -33,7 +33,7 @@ Create a file called **"config"** and put it under */etc/kubeturbo/*.
 {
 	"communicationConfig": {
 		"serverMeta": {
-			"turboServer": "<SERVER_ADDRESS>"
+			"turboServer": "https://<SERVER_IP_ADDRESS>:443"
```

Comment:
Should also change this note in the previous paragraph:
     **NOTE**: Turbonomic server address is "**IP address of your ops manager**".

It should be a HTTP Host, with host name and port.  Will it be true that OpsMgr will always use port 443 on HTTPS?   In my local deployment, the turboServer is **https://10.10.200.97:9400**.


 


For example

Reply:
Don't know if 9400 or 8400 are needed to be mentioned since people won't use them in a real turbo instance.

Reply:
Previously, OpsMgr runs on HTTP, and the default port for OpsMgr is 80, which can be omitted. But in a recent build, Opsmgr runs on HTTPS, and the default port is 443.  In this case, port number cannot be omitted in the test.

Reply:
I tested with https on port 443.  It is ok even if we do not specify the port number.
So we can commit the port number.